eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -60,14 +60,8 @@ jobs:
       - ${{ each artifact in parameters.Artifacts }}:
         - script: |
             mkdir -p $(Build.ArtifactStagingDirectory)/lint
-            mkdir -p $(Build.ArtifactStagingDirectory)/checkstyle
-            mkdir -p $(Build.ArtifactStagingDirectory)/spotbugs
-            cp sdk/${{parameters.ServiceDirectory}}/${{artifact.name}}/build/reports/lint-results.html \
-               $(Build.ArtifactStagingDirectory)/lint/${{artifact.name}}-lint-results.html
-            cp sdk/${{parameters.ServiceDirectory}}/${{artifact.name}}/build/reports/checkstyle/checkstyle.html \
-               $(Build.ArtifactStagingDirectory)/checkstyle/${{artifact.name}}-checkstyle.html
-            cp sdk/${{parameters.ServiceDirectory}}/${{artifact.name}}/build/spotbugs/debug.xml \
-               $(Build.ArtifactStagingDirectory)/spotbugs/${{artifact.name}}-spotbugs-debug.xml
+            cp sdk/${{parameters.ServiceDirectory}}/${{artifact.name}}/build/reports/lint-results-debug.html \
+               $(Build.ArtifactStagingDirectory)/lint/${{artifact.name}}-results-debug.html
```

Comment:
I think we need to revert this, otherwise our linter/checkstyle/spotbugs won't run

Reply:
I hadn't noticed, thank you for pointing it out